doozerlib/cli/release_gen_payload.py

```diff
@@ -155,6 +163,9 @@ def load_latest_builds(self):
         self._designate_privacy(latest_builds, images)
 
         mismatched_siblings = self._find_mismatched_siblings(latest_builds)
+        if self.runtime.assembly_type == assembly.AssemblyTypes.CUSTOM:
```

Comment:
Would like to mute this warning if no mismatched_siblings:

```suggestion
        if mismatched_siblings and self.runtime.assembly_type == assembly.AssemblyTypes.CUSTOM:
```